fhirstore/fhirstore.py

```diff
@@ -91,11 +89,24 @@ def bootstrap(self, resource: Optional[str] = None, show_progress: Optional[bool
                 [
                     ("identifier.system", ASCENDING),
                     ("identifier.value", ASCENDING),
+                ],
+                unique=True,
+                partialFilterExpression={
+                    "identifier.system": {"$exists": True},
+                    "identifier.value": {"$exists": True},
+                },
+            )
+            self.db[resource_type].create_index(
+                [
                     ("identifier.type.coding.system", ASCENDING),
                     ("identifier.type.coding.code", ASCENDING),
```

Comment:
Shouldn't this be:
```
("identifier.type.coding.0.code", ASCENDING),
("identifier.type.coding.0.system", ASCENDING),
```
?
See loader/src/reference_binder.py l32

Reply:
We'll remove the `0` in river also, we don't want to only check the first identifier 🙂